myia/prim/grad_implementations.py

```diff
@@ -321,6 +314,18 @@ def bprop_tagged(x, t, out, dout):
     return (casttag(dout, t), zeros_like(t))
 
 
+@register_bprop(primops.raise_)
+def bprop_raise_(x, out, dout):
+    """Backpropagator for primitive `raise_`."""
+    raise x
```

Comment:
Can this really be reached?

Reply:
I'm honestly not sure. Probably not, but it's the only expression that has the right type.